### PR TITLE
product pages added

### PR DESCRIPTION
**Pull request by tubakhan0415**

### CODE REVIEW

**Review by coder-faraz — CHANGES_REQUESTED** [review]
Remove junk files and push the code again

File: src/components/appBar/Appbar.jsx [hunk 1]
```diff
@@ -109,6 +109,7 @@ export default function ResponsiveAppBar() {
               fontSize: "1.3rem",
               borderRadius: "22px",
               marginRight: "10px",
+
```
Comment (codevamp-technologies) [inline src/components/appBar/Appbar.jsx:112]: discard these changes of Appbar.jsx

File: package.json [hunk 2]
```diff
@@ -25,6 +25,7 @@
     "react-router-dom": "^6.23.1",
     "redux": "^5.0.1",
     "redux-thunk": "^3.1.0",
+    "styled-components": "^6.1.12",
```
Comment (codevamp-technologies) [inline package.json]: do we need this package?

**Review by codevamp-technologies — CHANGES_REQUESTED** [review]

**Review by coder-faraz — APPROVED** [review]
looking fine

**Pull request merged**